graphene_django/templates/graphene/graphiql.html

```diff
@@ -100,22 +100,29 @@
     function updateURL() {
       history.replaceState(null, null, locationQuery(parameters));
     }
-    // Render <GraphiQL /> into the body.
-    ReactDOM.render(
-      React.createElement(GraphiQL, {
-        fetcher: graphQLFetcher,
+    // If there are any fragment parameters, confirm the user wants to use them.
+    var isReload = window.performance ? performance.navigation.type === 1 : false;
+    if (Object.keys(parameters).length
+      && !isReload
+      && !window.confirm("An untrusted query has been loaded, continue loading query?")) {
```

Comment:
Might be worth including an explanation of why running an untrusted query might be a bad idea, and what damage it could cause.

Also something like "if you're not sure click cancel"

This is the Facebook devtools warning for eg:

```
Stop!

This is a browser feature intended for developers. If someone told you to copy and paste something here to enable a Facebook feature or "hack" someone's account, it is a scam and will give them access to your Facebook account.

See https://www.facebook.com/selfxss for more information.
```